src/utilities/core-utils.test.ts

```diff
@@ -1,14 +1,89 @@
 import { CoreUtils } from "./core-utils";
 
 describe("CoreUtils", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region enumToArray
+    // -----------------------------------------------------------------------------------------
+
     describe("enumToArray", () => {
-        test.skip("TODO: Backfill tests to complete issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/issues/29", () => {});
+        test("when given enum, returns array of values", () => {
+            // Arrange
+            enum testEnum {
+                first = "First",
+                second = "Second",
+            }
+
+            // Act
+            const result = CoreUtils.enumToArray(testEnum);
+
+            // Assert
+            expect(result).toContain(testEnum.first);
+            expect(result).toContain(testEnum.second);
+        });
+
+        test("when given null enum, returns empty array", () => {
+            // Act
+            const result = CoreUtils.enumToArray(null);
+
+            // Assert
+            expect(result).toHaveLength(0);
+            expect(result).toBeInstanceOf(Array);
+        });
     });
 
+    // #endregion enumToarray
+
+    // -----------------------------------------------------------------------------------------
+    // #region getRandomEnum
+    // -----------------------------------------------------------------------------------------
+
     describe("getRandomEnum", () => {
-        test.skip("TODO: Backfill tests to complete issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/issues/29", () => {});
+        test("when given enum, returns random value", () => {
+            // Arrange
+            enum testEnum {
+                first = "First",
+                second = "Second",
+                third = "Third",
+            }
+            const expectedValues = [
+                testEnum.first,
+                testEnum.second,
+                testEnum.third,
+            ];
+
+            // Act
+            const result = CoreUtils.getRandomEnum(testEnum);
```

Comment:
There is no way of knowing if calling this 100 times would return the same value. I would call it 20+ times, add it to an array, sort the array and then make sure that first and last values are different.

Reply:
Wouldn't that still have the opportunity to be flaky though? Although unlikely, if all 20 (or 100...n) random calls returned the same enum, that would cause an erroneous failure

Reply:
@brandongregoryscott  @jstevenson  I made the loop with 50 iterations. You guys can decide whether I should keep it, change the number of iterations or completely remove it.